src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,155 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+
+/**
+ * Class contains variables and methods related to interactions with the user.
+ */
+public class Ui {
+    private static final String line = "____________________________________________________________";
```

Comment:
Maybe can name it in all caps "LINE" since its a constant!

Reply:
I'll do that! Thank you!